editor/find_in_files.cpp

```diff
@@ -705,12 +706,13 @@ void FindInFilesPanel::_notification(int p_what) {
 
 void FindInFilesPanel::_on_result_found(String fpath, int line_number, int begin, int end, String text) {
 	TreeItem *file_item;
+	Ref<Texture2D> remove_texture = get_editor_theme_icon(SNAME("RemoveEntry"));
```

Comment:
You can use the existing `Close` icon, which visually fits better with the rest of the editor theme. (This means you can remove `icons/RemoveEntry.svg`.)

Reply:
Fixed